dockers/docker-dhcp-server/cli/show/plugins/show_dhcp_server.py

```diff
@@ -110,7 +110,7 @@ def info(db, dhcp_interface, with_customized_options):
     for key in dbconn.keys("CONFIG_DB", "DHCP_SERVER_IPV4|" + dhcp_interface):
         entry = dbconn.get_all("CONFIG_DB", key)
         interface = key.split("|")[1]
-        table.append([interface, entry["mode"], entry["gateway"], entry["netmask"], entry["lease_time"], entry["state"]])
+        table.append([interface, entry["mode"], entry.get("gateway", ""), entry["netmask"], entry["lease_time"], entry["state"]])
```

Comment:
Hi @yue-fred-gao , is it better to have a "N/A" instead of empty string?

Reply:
This table style won't create confusion when a field is empty.
<img width="683" height="91" alt="image" src="https://github.com/user-attachments/assets/ce300894-685c-4215-9133-546ade05032e" />